src/main/java/org/codehaus/mojo/versions/ordering/NumericVersionComparator.java

```diff
@@ -31,7 +31,6 @@
  *
  * @author Stephen Connolly
  * @since 1.0-alpha-3
- * @deprecated
```

Comment:
This is not part of the fix...this should be made a different PR...

Reply:
That's fair.  I'll remove it.